### PR TITLE
[now-next] Provide a better error when failing to load routes-manifest

### PR DESCRIPTION
**Pull request by ijjk**
This updates the error message shown when we fail to load the `routes-manifest` which appears to be happening most often when an incorrect output directory is configured for a Next.js application

### CODE REVIEW

File: packages/now-next/src/utils.ts [hunk 1]
```diff
@@ -340,7 +340,7 @@ export async function getRoutesManifest(
 
   if (shouldHaveManifest && !hasRoutesManifest) {
     throw new Error(
-      `A routes-manifest.json couldn't be found. This could be due to a failure during the build`
+      `A routes-manifest.json couldn't be found. Is the correct output directory configured? This setting does not need to be changed in most cases. See more info here: https://err.sh/zeit/now/now-next-routes-manifest`
```
Comment (TooTallNate) [inline packages/now-next/src/utils.ts]:
```suggestion
      `A "routes-manifest.json" couldn't be found. Is the correct output directory configured? This setting does not need to be changed in most cases. More info: https://err.sh/zeit/now/now-next-routes-manifest`
```

**Review by TooTallNate — APPROVED** [review]
LGTM, but also left a suggestion.

File: packages/now-next/src/utils.ts [hunk 2]
```diff
@@ -340,7 +340,7 @@ export async function getRoutesManifest(
 
   if (shouldHaveManifest && !hasRoutesManifest) {
     throw new Error(
```
Comment (styfle) [inline packages/now-next/src/utils.ts:342]: @ijjk Let's change all the instances of `throw new Error()` to `throw new NowBuildError()` which supports the `link` property and also hides the stack trace.

Comment (ijjk) [reply packages/now-next/src/utils.ts:342]: Opened PR here https://github.com/zeit/now/pull/4167 to use `NowBuildError` instead

**Pull request merged**